glossary.yml

```diff
@@ -2454,6 +2454,10 @@
   am:
     term: የጥሪ ቁልል
     def: "የተተገበሩ  ንቁ ንዑስ አንቀጾች መረጃን የሚያከማች የውሂብ መዋቅር ።\t\n"
+  uk:
+    term: "стек викликів"
+    def: >
+      Структура даних, яка зберігає інформацію про підпрограми, що виконуються в даний момент.
```

Comment:
"Структура даних, яка зберігає інформацію про підпрограми, що виконуються в даний момент, включаючи параметри, локальні змінні та інструкції повернення."

I think it would be better what info does it usually contain

Reply:
+1

Reply:
@YehorBoiar @olexandr-konovalov I have updated, please review